examples/haskell-stack/stack.yaml.lock

```diff
@@ -0,0 +1,20 @@
+# This file was autogenerated by Stack.
```

Comment:
autogenererad?

Reply:
Bra tanke men lock-filer ska versionhanteras eftersom de låser in vilka versioner av dependencies som använda och hjälper därför till att garantera att samma kod köra av alla utvecklare/build pipelines :+1: